cloud/scope/powervs_cluster.go

```diff
@@ -1855,7 +1857,8 @@ func (s *PowerVSClusterScope) DeleteLoadBalancer() (bool, error) {
 				s.Info("VPC load balancer successfully deleted")
 				return false, nil
```

Comment:
I think if we return here, it won't proceed to the next, right?
Won't it loop the first item always and return from there?

Reply:
yes, I have changed the logic, PTAL!